File: web/src/__generated/model/static/NotifyConfig.ts
export interface NotifyConfig {
    readonly url: string;
    readonly headers: {readonly [key:string]: string};
    readonly body: string;
    readonly dailySummaryBody?: string | undefined;
    readonly dailySummaryCron?: string | undefined;
    readonly dailySummaryTimeZone?: string | undefined;
}

File: server/src/main/kotlin/com/coooolfan/xiaomialbumsyncer/model/NotifyConfig.kt
package com.coooolfan.xiaomialbumsyncer.model

import org.babyfish.jimmer.sql.Serialized

@Serialized
data class NotifyConfig(

    val url: String,

    val headers: Map<String, String>,

    val body: String,

    val dailySummaryBody: String?,

    val dailySummaryCron: String?,

    val dailySummaryTimeZone: String?,
)